File: Tareas 1/Actividad_Arreglos.cpp
#include <iostream>
int main() {
    //  Declaración
    int numeros[5];

    // Inicialización
    numeros[0] = 10;
    numeros[1] = 20;
    numeros[2] = 30;
    numeros[3] = 40;
    numeros[4] = 50;

    //  Asignación 
    numeros[2] = 100;  

    //   prueba en pantalla
    std::cout << "Contenido del arreglo:\n";
    for (int i = 0; i < 5; i++) {
        std::cout << "numeros[" << i << "] = " << numeros[i] << std::endl;
    }

    //  suma de dos elementos
    int suma = numeros[0] + numeros[2];  
    std::cout << "\nSuma de numeros[0] + numeros[2] = " << suma << std::endl;

    return 0;
}

File: Tareas 1/Actividad_Arreglos.java
public class ProgramaPrincipal {
    
    public static void main(String[] args) {
        // Declaración e inicialización de un arreglo de enteros con 6 valores
        int[] valores = {10, 20, 30, 40, 50, 60};
        
        // Modificación del valor en la posición 2 (índice 2) del arreglo
        valores[2] = 290;
        
        // Mostrar todo el contenido del arreglo en consola
        System.out.println("Contenido del arreglo:");
        for (int i = 0; i < valores.length; i++) {  
            // Imprime el índice y el valor almacenado en esa posición
            System.out.println("valores[" + i + "] = " + valores[i]);
        }
        
        // Se calcula la suma entre el primer elemento (índice 0) y el tercer elemento (índice 2)
        int suma = valores[0] + valores[2];
        
        // Se muestra el resultado de la suma en consola
      System.out.println("\nSuma de elementos: " + suma);
     }
}